src/shared/directives/conditional-show/domain/conditional-show.model.ts

```diff
@@ -61,6 +89,27 @@ export class ShowCondition {
     return [ condition.split(separator)[0], separator ];
   }
 
+  private static getConditions(formula: any): string {
+    let conditionList: string[] = [];
+    const newFormula = typeof formula === 'string' ? JSON.parse(formula) : formula;
+    if (!!formula) {
+      if (Array.isArray(newFormula)) {
```

Comment:
this two ifs can be merged to one to avoid too deep if statements.

Reply:
here in first if I'm checking whether formula field contains any value or not and in next if I'm evaluating the condition after I get value in formula field.
Still I have made small change in this.